geth/jail/console/console.go

```diff
@@ -0,0 +1,130 @@
+package console
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/robertkrimen/otto"
+	"github.com/status-im/status-go/geth/node"
+)
+
+const (
+	// EventConsoleLog defines the event type for the console.log call.
+	EventConsoleLog = "vm.console.log"
+
+	// EventConsoleWarn defines the event type for the console.debug call.
+	EventConsoleWarn = "vm.console.warn"
+
+	// EventConsoleDebug defines the event type for the console.debug call.
+	EventConsoleDebug = "vm.console.debug"
+
+	// EventConsoleError defines the event type for the console.error call.
+	EventConsoleError = "vm.console.error"
+)
+
+// Extension takes a giving extension function and writer and returns a standard otto.Function
+// callable by a otto vm.
+func Extension(w io.Writer, ext func(otto.FunctionCall, io.Writer) otto.Value) func(otto.FunctionCall) otto.Value {
+	return func(fn otto.FunctionCall) otto.Value {
+		return ext(fn, w)
+	}
+}
```

Comment:
You previously mentioned your dislike for global variables like the log writer. This function allows us to set a writer for the operation of a function.

I can turn the ext argyment into a type if it better helps your understanding. But if i dont do it here, will still have to do it in `registerHandlers` in `jail`.